src/vivarium_census_prl_synth_pop/components/person.py

```diff
@@ -7,6 +9,7 @@
 from vivarium.framework.randomness import RESIDUAL_CHOICE
 from vivarium.framework.utilities import from_yearly
 
+from vivarium_census_prl_synth_pop.components import HouseholdMigration
```

Comment:
Were you intending to type `self.household_migration` explicitly? I don't see where you use this import.

Reply:
leftover from when I was sub-componenting it